### DISCUSSION TITLE
Installing Wazuh after initial SO installation

### DISCUSSION THREAD

**Discussion by tjoachims** [Security-Onion-Solutions/securityonion#6337] [Unsupported Versions] [answered]
SO version: 2.3.80
Deployment: standalone
Base OS: Ubuntu Server 18.04.5 LTS

Issue: During initial setup of SO, I was given the option to select the various components and deselected Wazuh. Now that I'm months into using SO, I realize that I should have just had it installed since I would like to now use it. I've not found an easy way to install Wazuh besides having it selected as part of the initial SO installation but I would have to reinstall SO to take advantage of that. I've read on a reddit forum that simply installing Wazuh using the "sudo apt install wazuh" does not install (or configure) the necessary components as to work properly with SO as intended.

Question: is there a script or method to installing Wazuh on a existing SO install that did not initially install Wazuh as part of setup?

**Comment by dougburks** [comment, marked as answer]
Warning! This is totally untested and unsupported! If you try this, please try on a test system before trying in production!

There is a wazuh setting in the global pillar. You could try enabling it there and then running `sudo salt-call state.highstate`.

If all else fails, simply perform a new installation and make sure to select Wazuh.
